common/src/main/scala/explore/common/ConstraintSetObsQueries.scala

```diff
@@ -101,7 +103,9 @@ object ConstraintSetObsQueries {
       }
 
       object Observations {
-        type Nodes = ObsSummary
+        object Nodes {
+          type ConstraintSet = ConstraintsSummary
```

Comment:
Wherre is `ConstraintSummary` defined?

Reply:
It's in `explore.model`. It was already there, that part of the logic didn't change.

In this case, we are still decoding directly to our model type, since we are always querying the same set of fields.